src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private int radius;
+
+    public Circle(String color,int radius) {
```

Comment:
Format your code.

Reply:
I don't understand here. What do i need to format? Whitespace between parameters?